### DISCUSSION TITLE
Help me

### DISCUSSION THREAD

**Discussion by NeilTerrell12** [community/community#30029] [Other Feature Feedback, Questions, & Ideas]
I have no idea what I’m doing lol thanks for bring me on

**Comment by kumaratul60** [comment]
Do some creative activity

**Comment by jfullstackdev** [comment]
if you are asking about this Community, it's GitHub Community, helping one another for specific topics particularly about Git and GitHub and all related topics, plus the feedback for the GitHub features and bugs themselves,

if you are asking what is GitHub, it's a centralized space for collaborators, it's an efficient way to track changes of a file, mainly plaintext, because it was based on Git, and most of the time it's code, so it's associated with developers themselves

but GitHub is so innovative enough to add features even for non-dev, like hosting your site through GitHub Pages, and the Discussion tab itself that is a forum for everyone, you can even setup your own

**Comment by NikhilMath** [comment]
We can make a simple project together, I can teach you how to pull and push code. I think that would be good for ya

**Reply by kumaratul60** [reply]
sure, looking forward learn from you!

**Reply by NikhilMath** [reply]
my LinkedIn is my bio. I have some projects that are sooo easy that we can work on together. message me there
